src/base/bittorrent/trackerentry.cpp

```diff
@@ -59,8 +59,6 @@ bool TrackerEntry::isWorking() const
 #if (LIBTORRENT_VERSION_NUM < 10200)
     return nativeEntry().verified && nativeEntry().is_working();
```

Comment:
you should remove from here too.

Reply:
If I remove it then maybe for lower version of libtorrent nativeEntry is not be true and it will not say "working" in tracker status?